Command/DumpCommand.php

```diff
@@ -0,0 +1,250 @@
+<?php
+
+namespace Becklyn\MysqlDumpBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Process\Process;
+
+/**
+ * Handles the importing of the images from the command line
+ */
+class DumpCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure ()
+    {
+        $this
+            ->setName('becklyn:db:dump')
+            ->setDescription('Dumps the configured or given Databases using \'mysqldump\' to .sql files.')
```

Comment:
If you use double quotes, you can omit the escaping. Also: "databases" (with "d")

```php
->setDescription("Dumps the configured or given databases using 'mysqldump' to .sql files.")
```